ROADMAP.md

```diff
@@ -1,40 +1,64 @@
 # Roadmap
 
 This document defines a high level roadmap for Rook development.
-The dates below are subject to change but should give a general idea of what we are planning.
+Any dates listed below are subject to change but should give a general idea of what we are planning.
 We use the [milestone](https://github.com/rook/rook/milestones) feature in Github so look there for the most up-to-date and issue plan.
 
-## Rook 0.8
-
-* Multiple storage backends
-  * Full design
-  * Refactor code base and repositories to enable
-  * Consider support for Minio, potentially early support for other backends time permitting
-* Run on arbitrary PVs
-* Remove Rook API and CLI
-* Run with Least Privileged design
-  * Improved support for OpenShift
-* Support Kubernetes 1.7+ only
-* Ceph features and improvements
-  * Disk management (1 OSD per pod, adding/removing disks)
-  * Placement group balancer support (ceph-mgr module)
-  * Mon reliability (one mon per node, failing over too fast, etc.)
-
 ## Rook 0.9
```

Comment:
There are a LOT of items in the 0.9 milestone, I am very open to shifting some of them to future milestones.  We can also consider a 0.10 milestone before we get to 1.0.

Reply:
There are many items in this list that don't have owners yet, which puts them at risk for this milestone. This makes me wonder if there is a better way to represent the roadmap. The key is that we still reserve the right to add/remove items to the list and it is heavily dependent on the amount of community involvement. What if we added a comment to that effect at the top of the roadmap? I'd suggest we optimistically keep items in the 0.9 milestone until we get closer to the release date before we punt them to the next milestone.

Reply:
i think this makes sense, i will add some language around this and we'll review.  this fits into @bassam's suggestion about moving some items up if we can find an owner for them.  perhaps this roadmap can help to serve as a "call for contributors" to take ownership for some of these features, with the caveat that some items may be pushed to later milestones if owners can't be found.  i'll take a stab at that.